docs/4.0.0/user-guide/migrating-to-4.0.0.md

```diff
@@ -77,8 +77,6 @@ To make the upgrade process easier, we've introduced the [`indent-legacy`](/docs
 
 ## <a name="config-validation"/> Unrecognized properties in config files now cause a fatal error
 
-**Note:** This feature is a work in progress and has not been released yet.
```

Comment:
No, it's been released now. I added that line intentionally as part of https://github.com/eslint/eslint/commit/734846b1557a45eb43286c4da2bdc1e13b89d1c6.